apps/dashboard/src/components/integrations/components/integration-general-settings.tsx

```diff
@@ -42,6 +56,52 @@ export function GeneralSettings({ control, mode, hidePrimarySelector, disabledPr
           </FormItem>
         )}
       />
+      {isForInAppStep && (
+        <FormField
+          control={control}
+          name="removeNovuBranding"
+          render={({ field }) => {
+            const isFreePlan = subscription?.apiServiceLevel === ApiServiceLevelEnum.FREE;
+            const disabled = isFreePlan || isLoadingSubscription;
+            const value = disabled ? false : field.value;
+
+            const switchControl = <Switch disabled={disabled} onCheckedChange={field.onChange} checked={value} />;
+
+            return (
+              <FormItem className="flex items-center justify-between gap-2">
+                <FormLabel
+                  className="text-xs"
+                  htmlFor="active"
+                  tooltip='Hide "Powered by Novu" branding from your <Inbox />'
+                >
+                  Remove "Powered by Novu" branding
+                </FormLabel>
+                <FormControl>
+                  {disabled ? (
+                    <HoverCard openDelay={100} closeDelay={100}>
+                      <HoverCardTrigger asChild>{switchControl}</HoverCardTrigger>
+                      {disabled && (
```

Comment:
```suggestion
                      {disabled && (
```
remove this part as it will be always true in this case because of the check above